examples/jsm/webxr/XRButton.js

```diff
@@ -1,6 +1,6 @@
 class XRButton {
 
-	static createButton( renderer ) {
+	static createButton( renderer, sessionInit = {} ) {
```

Comment:
I can't merge this PR until @mrdoob approves since I'm not why `sessionInit` wasn't added in the first place. Looking at the code, the module request multiple optional features at once. Maybe this was done in order to keep the interface simple. Adding `sessionInit` means devs have to know how the `options` of `requestSession()` is structured.